code/game/objects/effects/temporary_visuals/temporary_visual.dm

```diff
@@ -73,8 +73,8 @@
 	holder = hud_list[XENO_TACTICAL_HUD]
 	if(!holder)
 		return
-	holder.icon = 'icons/Marine/marine-items.dmi'
-	holder.icon_state = "detector_blip"
+	holder.icon = 'icons/effects/blips.dmi' // wasn't working, dunno if we should keep it
+	holder.icon_state = "close_blip_hostile"
```

Comment:
?

Reply:
"close_blip_hostile" used to be a ”detector_blip" and for this reason this hud wasn't working properly

Reply:
Probably should delete the comment